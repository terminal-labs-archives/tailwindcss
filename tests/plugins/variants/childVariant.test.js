import { quickVariantPluginTest } from '../../util/run'

quickVariantPluginTest('childVariant').toMatchSnapshot()
