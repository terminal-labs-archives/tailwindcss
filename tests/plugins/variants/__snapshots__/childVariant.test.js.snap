// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the 'childVariant' plugin 1`] = `
"
.\\*\\:flex > * {
  display: flex;
}
"
`;
